### PR TITLE
Past prod style tweak

### PR DESCRIPTION
**Pull request by gorajski**
Tiny styling adjustment to /past-productions page to make matching dates to productions more understandable.  Achieved by reducing spacing between each production and its corresponding date.

### CODE REVIEW

File: _layouts/production.html [hunk 1]
```diff
@@ -2,87 +2,112 @@
 layout: default
 ---
 
+<!-- This bit of Liquid counts the number of images, image_count, for use in flow control below -->
```
Comment (hollabaq86) [inline _layouts/production.html]: though the comment is appreciated, please go ahead and delete from your branch.  We can infer what this code does from comments below (show Carousel comment, etc).

**Review by hollabaq86 — COMMENTED** [review]
real small tweak to the code please, I'm being nit-picky.

**Comment by gorajski** [comment]
I removed the commented line from production.html.  Unless there are no more comments, ready to merge.

**Review by hollabaq86 — APPROVED** [review]

**Comment by hollabaq86** [comment]
Hey Nick, I just merged Patrick's PR and now there are merge conflicts with your PR. Would you mind taking a look?

**Pull request merged**